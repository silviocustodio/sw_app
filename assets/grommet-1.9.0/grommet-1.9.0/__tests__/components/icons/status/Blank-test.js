// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Blank from '../../../../src/js/components/icons/status/Blank';

describe('Blank', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Blank />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('has correct className rendering', () => {
    const component = renderer.create(
      <Blank className='testing' />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
