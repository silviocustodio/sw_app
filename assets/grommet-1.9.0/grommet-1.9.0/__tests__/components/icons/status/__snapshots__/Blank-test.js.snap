// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blank has correct className rendering 1`] = `
<svg
  aria-label="Blank"
  className="grommetux-status-icon grommetux-status-icon-blank testing"
  role="img"
  version="1.1"
  viewBox="0 0 24 24"
/>
`;

exports[`Blank has correct default options 1`] = `
<svg
  aria-label="Blank"
  className="grommetux-status-icon grommetux-status-icon-blank"
  role="img"
  version="1.1"
  viewBox="0 0 24 24"
/>
`;
